setup/setup.sh

```diff
@@ -174,59 +190,80 @@ func_python_deps(){
   unzip -o requiredfiles.zip
 
   # Prepare Wine Directories
-  echo ' [*] Preparing Wine Directories'
-  mkdir -p ~/.wine/drive_c/Python27/Lib/
+  echo ' [*] Preparing Wine Directories (#1)'
+  mkdir -p ~/.wine/drive_c/Python27/Lib/site-packages/
   cp distutils -r ~/.wine/drive_c/Python27/Lib/
   cp tcl -r ~/.wine/drive_c/Python27/
   cp Tools -r ~/.wine/drive_c/Python27/
 
   # Install Setup Files
-  echo ' [*] Installing Wine Python Dependencies'
-  wine msiexec /i python-2.7.5.msi
-  wine pywin32-218.win32-py2.7.exe
-  wine pycrypto-2.6.win32-py2.7.exe
+  echo ' [*] Installing Wine Python Dependencies (#1)'
+  arg=""
+  [[ "$silent" ]] && arg="/q"
+  wine msiexec /i python-2.7.5.msi $arg
+  for FILE in pywin32-218.win32-py2.7.exe pycrypto-2.6.win32-py2.7.exe; do
+    if [[ "$silent" ]]; then
+      unzip -q -o $FILE
+      cp -rf PLATLIB/* ~/.wine/drive_c/Python27/Lib/site-packages/
+      rm -rf "PLATLIB/"
+    else
+      wine $FILE
+    fi
+  done
   if [ -d "/opt/pyinstaller-2.0/" ]; then
     echo ' [*] PyInstaller Already Installed... Skipping.'
   else
     sudo unzip -d /opt pyinstaller-2.0.zip
-    sudo chmod 755 -R /opt/pyinstaller-2.0/
+    sudo chmod 0755 -R /opt/pyinstaller-2.0/
   fi
 
   if [ $(uname -m) == 'x86_64' ]; then
 
     # Prepare Wine Directories
-    echo ' [*] Preparing Wine64 Directories'
+    echo ' [*] Preparing Wine64 Directories (#2)'
     mkdir -p ~/.wine64/drive_c/Python27/Lib/
```

Comment:
Would this line also need to have the same modification you created on line 194? (mkdir -p ~/.wine/drive_c/Python27/Lib/site-packages/).  It looks like right below you're referencing the site-packages/ directory, but it wasn't created yet (but just wanted to verify).